roles/openshift_health_checker/library/aos_version.py

```diff
@@ -26,6 +27,13 @@
 except ImportError as err:
     IMPORT_EXCEPTION = err
 
+# NOTE: yum does not work with Python 3, but since the openshift-ansible code
+# base is tested against both Python 2 and Python 3, we write the code below for
+# compatibility with the latter:
+PY3 = sys.version_info[0] == 3
+if PY3:
+    basestring = str
```

Comment:
The idea is borrowed from `six`, with an adaptation to define `basestring` instead of `string_types`, so avoiding introducing an extra name.

This is only needed to make the unit tests pass in py3, even though in practice one cannot use py3 with this check because of the dependency on `yum` (py2-only).